src/data/proxy/PostStore.java

```diff
@@ -48,7 +48,6 @@ public void write(Post post) {
      */
     public List<Post> getPostsByUser(String user) {
         Predicate<Post> predicate = new Predicate<Post>() {
-            @Override
```

Comment:
I had removed these @Overrides in the work I was doing. Good to see you caught it :+1: